third_party/move/move-compiler-v2/transactional-tests/tests/no-v1-comparison/positional_fields/common_access.move

```diff
@@ -0,0 +1,18 @@
+//# publish
+module 0x42::test {
+    enum Foo has drop {
+        A(u8),
+        B(u8),
+    }
+
+    fun common_access(x: Foo): u8 {
+        x.0
+    }
+
+    fun test_invalid(): u8 {
```

Comment:
It would be nice not to use "invalid" in the context where it is indeed valid. I was trying to figure out what is invalid about this!

Reply:
done